src/Overlay.tsx

```diff
@@ -179,6 +193,10 @@ const Overlay = React.forwardRef<HTMLDivElement, OwnOverlayProps>(
       )
     }, [anchorSide, slideAnimationDistance, slideAnimationEasing, visibility])
 
+    // To be backwards compatible with the old Overlay, we need to set the left prop if x-position is not specified
+    const leftPosition: React.CSSProperties =
+      left === undefined && right === undefined ? {left: 0} : left ? {left: `${left}px`} : {left}
```

Comment:
I think this can be simplified to something like this:

```suggestion
  const leftPosition: React.CSSProperties = left === undefined && right === undefined ? {left: 0} : {left}
```

Numbers are assumed to be `px` when passed to the `style` prop.

Reply:
Perfect!